.github/workflows/publish_main_snapshot.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.SIGNING_KEY_ID }}
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.SIGNING_PASSWORD }}
         run: ./gradlew $GRADLE_OPTIONS :newrelic-scala-api:publish :newrelic-scala-cats-api:publish :newrelic-cats-effect3-api:publish :newrelic-scala-zio-api:publish
-      - name: Publish snapshot apis for k2
```

Comment:
I've been removing references to K2 (since that is the name of a company that no longer exists) and replacing them with "Security agent" so it's less confusing for future code spelunkers.